lnurl/explainer.md

```diff
@@ -0,0 +1,352 @@
+# LNURL
```

Comment:
wanted to preview on the bolt.fun site but the yaml is missing so its not rendering.

Reply:
Yea I was doing the content only in this higher dir first and then after planned to move it into bolt.fun dir with proper formatting and everythin